recipes/theta2/meta.yaml

```diff
@@ -9,6 +9,9 @@ source:
   sha256: 474452fa87a1ca0228137693bfede3200b91e4e166b2216c4bd24f8b270c1dea
 
 requirements:
+  build:
```

Comment:
afaik, only the compiler stuff should be in build, the rest in host

Reply:
Oops, that'll teach me to edit late at night. I guess you merged this already, though.

Reply:
Jupp, fixed it before.

Reply:
You...don't sleep do you.